_posts/2020-04-03-what-does-appsync-do.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: What does AWS AppSync do ?
+date: 2020-04-03
+tags: AWS AppSync Cloudformation
+author: Shermayne Lee
+---
+
+# Overview
+
+AWS AppSync and GraphQL have gained popularity in recent years. However it can be confusing and difficult to start for developers new to graphQL concept.
+
+AWS AppSync is a managed GraphQL platform by AWS for serverless implementation. It can translate graphQL requests from client, get the data from AWS Resources and pass the data to the client.
+
+# Introduction to AppSync
+
+AWS Appsync is the service that helps your application development process simple by providing the ability to create flexible API. It can be the central hub that connects all your data sources securely.
+
+AWS AppSync is designed mainly for the use of GraphQL API and one of the main benefits of AppSync is letting your application get the exact data that you need. Additionally for mobile and web applications, it provides the ability to access data locally when devices go offline and synchronize application data in real-time when devices are back online.
```

Comment:
If you use the content in my comment above you can probably remove the part "AWS AppSync is designed mainly for the use of GraphQL API and"